packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -2915,6 +2916,118 @@ testWidgets('SnackBarAction backgroundColor works as a Color', (WidgetTester tes
 
     expect(material.clipBehavior, Clip.antiAlias);
   });
+
+ testWidgets('Tap on button behind snack bar defined by width',
+      (WidgetTester tester) async {
+    tester.view.physicalSize = const Size.square(200);
+    tester.view.devicePixelRatio = 1;
+    addTearDown(tester.view.resetPhysicalSize);
+    addTearDown(tester.view.resetDevicePixelRatio);
```

Comment:
Is it necessary for this test to change the physical size and pixel ratio ?

Reply:
It's to make sure I don't click on the snackbar.